src/SourceBuild/content/repo-projects/aspnetcore.proj

```diff
@@ -21,12 +21,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <RepositoryReference Include="arcade" />
-    <RepositoryReference Include="source-build-externals" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
-    <RepositoryReference Include="runtime" />
-    <RepositoryReference Include="msbuild" />
+    <RepositoryReference Include="nuget-client" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
     <RepositoryReference Include="roslyn" />
```

Comment:
We should keep the top level dependencies explicit for each repo. So this should depend on runtime, arcade, SBE etc. Transitive needs can be pulled in via new automation.

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/608e3adab65aca27adcc828a17383c6b401466dc